src/snowflake/snowpark/dataframe.py

```diff
@@ -3026,6 +3026,12 @@ def copy_into_table(
         transformations: Optional[Iterable[ColumnOrName]] = None,
         format_type_options: Optional[Dict[str, Any]] = None,
         statement_params: Optional[Dict[str, str]] = None,
+        is_iceberg: bool = False,
+        external_volume: Optional[str] = None,
+        catalog: Optional[str] = None,
+        base_location: Optional[str] = None,
+        catalog_sync: Optional[str] = None,
+        storage_serialization_policy: Optional[str] = None,
```

Comment:
I'm hesitating whether we should put these into a dictionary with parameter `iceberg_config` so we group them altogether. Same for save_as_table. What do you think?

Reply:
That seems alright to me. There isn't any precedent to follow from spark since our io is already quite different. That might be a little bit more boilerplate for the user, but these values aren't very variable so they could probably just make a constant dict of the settings and use it whenever there is a save_as_table or copy_into call.

Reply:
I've refactored based on this suggestion.